modules/serverless-nodejs-template.adoc

```diff
@@ -6,7 +6,7 @@
 [id="serverless-nodejs-template_{context}"]
 = Node.js function template structure
 
-When you create a Node.js function using the `kn` CLI, the project directory looks like a typical Node.js project, with the exception of an additional `func.yaml` configuration file.
+When you create a Node.js function using the `kn` CLI, the project directory looks like a typical Node.js project. The only exception is the additional `func.yaml` configuration file, which is used for specifying the image.
```

Comment:
```suggestion
When you create a Node.js function using the `kn` CLI, the project directory looks like a typical Node.js project. The only exception is the additional `func.yaml` file, which is used to configure the function.
```
The func.yaml file is responsible for a lot more than simply specifying the image. It is responsible for the entire function configuration.